site/src/sphinx/client-retry.rst

```diff
@@ -211,7 +214,27 @@ it is timed out by the per-attempt timeout.
                                            10,000ms (ResponseTimeoutException)
     @endditaa
 
+In above example, every attempt is made before it is timed out because the :api:`Backoff` is disabled.
+However, What if a :api:`Backoff` is enabled and the moment of trying next attempt is after the point of
+occurring :api:`ResponseTimeoutException`? The :api:`RetryingClient` do not schedule for the next attempt and
+returned with the latest :api:`Response` from the server immediately. Consider following example:
```

Comment:
- returned with the latest ..immediately. -> finish the retry session immediately with the last ...
- following example -> the following example